go/vt/mysqlctl/schema.go

```diff
@@ -21,7 +21,7 @@ func (mysqld *Mysqld) GetSchema(dbName string, tables, excludeTables []string, i
 	sd := &proto.SchemaDefinition{}
 
 	// get the database creation command
-	qr, fetchErr := mysqld.FetchSuperQuery("SHOW CREATE DATABASE " + dbName)
+	qr, fetchErr := mysqld.FetchSuperQuery("SHOW CREATE DATABASE IF NOT EXISTS " + dbName)
```

Comment:
nice fix :), it is required for this example? It looks like the vttablet-pod-template creates the database on vttablet instantiation.